.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -779,6 +787,58 @@ stages:
         - template: ./templates/.builder-release-template.yaml
           parameters:
             artifactName: 2004-fips-gen2-containerd
+  - stage: build_vhd_2204_fips_containerd
+    dependsOn: []
+    condition: eq('${{ parameters.build2204fipscontainerd }}', true)
+    jobs:
+    - job: build
+      timeoutInMinutes: 180
+      steps:
+        - bash: |
+            echo '##vso[task.setvariable variable=DRY_RUN]${{parameters.dryrun}}'
+            echo '##vso[task.setvariable variable=OS_SKU]Ubuntu'
+            echo '##vso[task.setvariable variable=OS_VERSION]22.04'
+            echo '##vso[task.setvariable variable=IMG_PUBLISHER]Canonical'
+            echo '##vso[task.setvariable variable=IMG_OFFER]0001-com-ubuntu-server-jammy'
+            echo '##vso[task.setvariable variable=IMG_SKU]22_04-lts'
```

Comment:
can we also specify IMG_VERSION=latest, as: https://github.com/Azure/AgentBaker/blob/e487581991aaf88a0b37771c9ebd8c7c5406c04e/.pipelines/.vsts-vhd-builder-release.yaml#L900

it is not necessary since packer will default to latest, however, for an effort I am currently working on for reproducible VHDs, I want the variable to be specified here.

Reply:
sure